packages/graphback-cli/src/templates/configTemplates.ts

```diff
@@ -0,0 +1,52 @@
+import { writeFileSync } from 'fs';
+
+/**
+ * Different database choices
+ */
+export const defaultDB = {
+  name: 'postgres',
+  config: (projectName: string) => {
+    return {
+      'user': 'postgresql',
+      'password': 'postgres',
+      'database': `${projectName}`,
+      'host': '127.0.0.1',
+      'port': '5432'
+    }
+  }
+}
+
+const pgDockerCompose = (projectName: string) => {
```

Comment:
I really do not like this. I think we should have them as files. No strong opinion, but it will be much cleaner approach and we can actually test them .current approach do not allow us to test it